avocado/plugins/jobscripts.py

```diff
@@ -15,7 +15,7 @@ class JobScripts(JobPre, JobPost):
     description = 'Runs scripts before/after the job is run'
 
     def __init__(self):
-        self.log = logging.getLogger("avocado.app")
+        self.log = LOG_UI
```

Comment:
even more true here, I can imagine we'll want to put each concurrent job's output into a different logger, so removing the loggers just to re-introduce it later looks unnecessary.